api/server/router/local/info.go

```diff
@@ -54,26 +52,27 @@ func (s *router) getInfo(ctx context.Context, w http.ResponseWriter, r *http.Req
 	return httputils.WriteJSON(w, http.StatusOK, info)
 }
 
+func buildOutputEncoder(w http.ResponseWriter) *json.Encoder {
```

Comment:
I grpe'ed for other uses of `outStream.Write(nil)` and only found one that didn't set the encoder or content type.

If you think this will be used in the future I can move it now, but it doesn't appear to be used at this time.